kahuna/public/js/search/query.js

```diff
@@ -44,6 +44,13 @@ query.controller('SearchQueryCtrl',
         {label: 'Past week',     value: pastWeek}
     ];
 
+    ctrl.orderByOptions = [
+        {label: 'Upload Time (desc)'},
+        {label: 'Upload Time (asc)', value: 'uploadTime'},
```

Comment:
Joanna Ruck asked for it for pics of the day as they search, go to the bottom, and then go up.